docs/sourcemap-api.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[sourcemap-api]]
+== Sourcemap API
```

Comment:
I wonder if everyone who reads this will know what source maps are and how they are used. If we expect this to be a new concept to some users, then you should add a couple of sentences that describe what a sourcemaps are and how they are used for debugging.